pytorch_toolkit/action_recognition_2/model_templates/gesture-recognition/s3d-rgb-mobilenet-v3-stream-jester/template.yaml

```diff
@@ -38,7 +39,7 @@ output_format:
   openvino:
     default: true
     input_format: RGB
-compression: null
+optimisations: ~
```

Comment:
both of them is defined in yaml standard. I like `~` because it looks like a special symbol rather than string/text. But I have no strong argument to use `~` and not to use `null`

Reply:
Ok! But IMHO the choice should be consistent -- so, if we use `~`, we should not use `null`.